packages/cli/src/commands/logs/command.ts

```diff
@@ -12,50 +16,25 @@ export const logsCommand: Command = {
   ],
   options: [
     {
-      name: 'follow',
```

Comment:
Instead of removing these outright, can you mark them as deprecated so we are able to link a helpful message?

As an example, we do something similar in the [`deploy` command](https://github.com/vercel/vercel/blob/c75415ae3c46cf62790c2541ef0d602e3c534dad/packages/cli/src/commands/deploy/index.ts#L167-L176). 

We'll likely need a changelog that talks about the major version bump somewhere, which this message could link to.